.npmignore

```diff
@@ -0,0 +1,9 @@
+.editorconfig
+.eslintrc
+.gitignore
+.istanbul.yml
+.jscs.json
+.travis.yml
+CHANGELOG.md
+Makefile
+test.js
```

Comment:
Tests and linting config must be published, so that `npm explore foo && npm install && npm test` passes.

Including the changelog is critical so people can read about version changes offline.

The only files that would thus be exclude-able are the CI config and the Makefile, which are insignificant in size.

Reply:
I had never looked into `npm explore` before, but that's interesting. Makes perfect sense. Thanks for letting me know!